spec/02-integration/02-cmd/17-drain_spec.lua

```diff
@@ -0,0 +1,269 @@
+local helpers = require "spec.helpers"
+local http = require "resty.http"
+
+local cp_status_port = helpers.get_available_port()
+local dp_status_port = 8100
+
+local function get_status_no_ssl_verify()
+  local httpc = http.new()
+
+  local ok, err = httpc:connect({
+      scheme = "https",
+      host = "127.0.0.1",
+      port = dp_status_port,
+      ssl_verify = false,
+  })
+  if not ok then
+      return nil, err
+  end
+
+  local res, err = httpc:request({
+      path = "/status/ready",
+      headers = {
+          ["Content-Type"] = "application/json",
+      }
+  })
+
+  if not res then
+    return nil, err
+  end
+
+  return res.status
+end
+
+for _, strategy in helpers.each_strategy() do
+  if strategy ~= "off" then
+    -- skip the "off" strategy, as dbless has its own test suite
```

Comment:
Where is the dbless test suite? If this is used with k8s, it needs to support dbless

Reply:
add DB-less test case